packages/taquito-core/README.md

```diff
@@ -0,0 +1,3 @@
+# Taquito Core package
+
+`@taquito/core` (under development) plan to extract error classes and interfaces used across packages to avoid circular dependencies in taquito. 
```

Comment:
I suggest we remove the `(under development)` and `plan`. Even when the work is partially done, it serves its purpose, so something like:

```
This package contains classes and interfaces used by other Taquito packages, to avoid circular dependencies.
```